ragaai_catalyst/guard_executor.py

```
@@ -66,32 +125,144 @@ def __call__(self,messages,prompt_params,model_params,llm_caller='litellm'):
-            return response['data']['alternateResponse'],llm_response,response
+        alternate_op_response,output_deployment_response = self.execute_output_guardrails(llm_response)
+        if output_deployment_response and output_deployment_response['data']['status'].lower() == 'fail':
+            return alternate_op_response,llm_response,output_deployment_response
         else:
-            return None,llm_response,response
+            return None,llm_response,output_deployment_response
 
+    def set_variables(self,prompt,prompt_params):
+        for key in self.field_map:
+            if key not in ['prompt', 'response']:
+                if self.field_map[key] not in prompt_params:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)